src/kytos/interfaceInfo.vue

```diff
@@ -88,12 +115,48 @@ export default {
     change_plotRange(range) {
         this.plotRange = range
         this.update_chart()
-    }
+    },
+    get_metadata() {
+      if(this.content === undefined) return
+      this.metadata_items = this.content.metadata
+    },
+    get_next_state() {
```

Comment:
@Alopalao make sure to also call this in the `watch` content otherwise this might result in an unexpected state if users select other interface. Could you double check this?

Reply:
Fixed on commit [e7e913377d6499a719a59007ef41d415848a3674](https://github.com/kytos-ng/ui/pull/26/commits/e7e913377d6499a719a59007ef41d415848a3674)

Reply:
@Alopalao there seems to still be an issue with the state in a specific case, for example:

1) Select an interface let's say `00:00:00:00:00:00:00:01:1` (s1-eth1), click on the disable button, it'll update the panel and attributes correctly
2) Without closing the panel, click on the same switch, and if you select the same interface again, you'll see the attributes will have the old `enabled` values. 

Could you double check this case?

Reply:
Looked like `content.interfaces` from `k-info-panel/switch_info` is not getting updated. I created an topology PR [#89](https://github.com/kytos-ng/topology/pull/89) so it updates correctly. Maybe a temporal solution while I look from where `k-info-panel/switch_info` gets its data.

Reply:
Looks like the `content.interface` from `k-info-panel/switch_info` does not update. I created a topology PR [#89](https://github.com/kytos-ng/topology/pull/89) which solves the problem. This is maybe a temporal solution until I find where does `k-info-panel/switch_info` get its interface details.